src/librvnpal/src/posix/mapping.c

```diff
@@ -1,5 +1,11 @@
 #if defined(__unix__) || defined(__APPLE__)
 
+#ifdef __APPLE__
+#define MMAP mmap
```

Comment:
We can probably just do `mmap64` ->`mmap` here, and avoid `MMAP`

Reply:
as a precocious i would like to leave it this way, as :
```
void *mmap    (void *addr, size_t len, int prot, int flags, int fildes, off_t off);
void *mmap64(void *addr, size_t len, int prot, int flags, int fildes, off64_t off); 
``` 
in [here](https://www.mkssoftware.com/docs/man3/mmap.3.asp) it states : `The mmap64() function is a part of the large file extensions. `